app.cantara.Cantara.yml

```
@@ -0,0 +1,41 @@
+app-id: app.cantara.Cantara
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.freepascal
+command: cantara
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home
+#icon: app.cantara.Cantara.png
+#rename-desktop-file: cantara.desktop
+modules:
+  - name: qt5pas
+    buildsystem: qmake
+    config-opts:
+      - -after
+      - target.path=/app/lib
+    sources:
+      - type: shell
+        commands:
+          - cp -r /usr/lib/sdk/freepascal/share/lazarus/lcl/interfaces/qt5/cbindings/. .
+  - name: cantara
+    sources:
+      - type: archive
+        url: https://github.com/reckel-jm/cantara/archive/refs/tags/v2.3.2.tar.gz
+        sha256: 9ee5c0bee8da25be1f4918c362b3db069028b1a24da224d0caded2cd51d7e9be
+    buildsystem: simple
+    build-commands:
+      - |
+        . /usr/lib/sdk/freepascal/enable.sh
+        lazbuild --lazarusdir="$LAZARUS_DIR" -B --bm="Release" --ws="qt5" Cantara.lpi
+      - install -D cantara -t $FLATPAK_DEST/bin
+      - cp -r languages $FLATPAK_DEST/bin/
+      - install -D icons/64.png $FLATPAK_DEST/share/icons/hicolor/64x64/app.cantara.Cantara.png
+      - install -D icons/128.png $FLATPAK_DEST/share/icons/hicolor/128x128/app.cantara.Cantara.png
+      - install app.cantara.Cantara.png $FLATPAK_DEST/share/icons/app.cantara.Cantara.png
+      - install -D app.cantara.Cantara.desktop -t $FLATPAK_DEST/share/applications
+      - install -D app.cantara.Cantara.metainfo.xml $FLATPAK_DEST/share/app-info/xmls/app.cantara.Cantara.xml
```

This is not were it gets installed.

```suggestion
      - install -D app.cantara.Cantara.metainfo.xml -t $FLATPAK_DEST/share/metainfo
```